wqflask/tests/wqflask/show_trait/test_show_trait.py

```diff
@@ -1,24 +1,11 @@
 """test for wqflask/show_trait/test_show_trait.py"""
-
-
 import unittest
 from unittest import mock
 from wqflask import app
-from wqflask.show_trait.show_trait import check_if_attr_exists
-from wqflask.show_trait.show_trait import requests
-
-from wqflask.show_trait.show_trait import get_ncbi_summary
-
-from wqflask.show_trait.show_trait import has_num_cases
-
-from wqflask.show_trait.show_trait import get_table_widths
-
-from wqflask.show_trait.show_trait import get_categorical_variables
-from wqflask.show_trait.show_trait import get_trait_units
-
-from wqflask.show_trait.show_trait import get_nearest_marker
-
-from wqflask.show_trait.show_trait import get_genotype_scales
+from wqflask.show_trait.show_trait import (check_if_attr_exists, get_ncbi_summary,
+                                           has_num_cases, get_table_widths,
+                                           get_categorical_variables,
+                                           get_trait_units, get_nearest_marker, get_genotype_scales, requests)
```

Comment:
**nitpick:** By group I never reallly meant this

AFAICT, PEP 8 recommends:
```
from lib import A
from lib import B
from lib import C
```
instead of
```
from lib import (A, B, C)
```

:smile: